src/components/SendValueTransferForm.vue

```diff
@@ -35,20 +35,47 @@
         <AppendUnit slot="append" :static-unit="WIT_UNIT.NANO" />
       </el-input>
     </el-form-item>
-    <el-form-item :label="$t('timelock')" prop="timelock">
-      <el-date-picker
-        v-model="form.timelock"
-        type="datetime"
-        placeholder="Select date and time"
-        tabindex="5"
-        :default-value="new Date()"
-        value-format="timestamp"
-      />
-    </el-form-item>
+    <transition name="slide">
+      <el-form-item
+        v-if="isAdvancedVisible"
+        :label="$t('timelock')"
+        prop="timelock"
+      >
+        <el-date-picker
+          v-model="form.timelock"
+          type="datetime"
+          :placeholder="$t('select_date')"
+          tabindex="5"
+          :default-value="new Date()"
+          value-format="timestamp"
+        />
+      </el-form-item>
+    </transition>
     <p v-if="createVTTError" class="error">{{ createVTTError.message }}</p>
     <div class="submit">
       <el-button
-        tabindex="5"
+        v-if="isAdvancedVisible"
```

Comment:
Add css rule `width: fit-content` to avoid wrong behavior clicking outside of the button